scripts/graph.sh

```diff
@@ -19,4 +19,15 @@ DIR=$(dirname -- "${BASH_SOURCE[0]}")
         "\"\(.from)\" -> \"\(.to)\""
     '
     echo '}'
-} | dot -Tpng > "$DIR"/../packages.png
+    # normalize
+} | dot -Tcanon >packages-graph.dot
+dot -Tpng <packages-graph.dot >"$DIR"/../packages-graph.png
+
+dot -Tsvg <packages-graph.dot >"$DIR"/../packages-graph.svg
+
+if acyclic packages-graph.dot | dot -Tcanon >packages-sans-cycles.dot; then
+    echo "No cycles in 'dependencies' of packages."
+else
+    echo "Cycles detected. These lines appear only in the original graph and not the acyclic variant:"
+    comm -23 <(sort packages-graph.dot) <(sort packages-sans-cycles.dot)
```

Comment:
I had to look up `comm -23` and `<(...)` [process substitution syntax](https://unix.stackexchange.com/a/596893). I was about to ask for a comment, but the `echo` line above says what it's doing. Pretty clearly not posix `sh`, but it says `#!/bin/bash` on the tin. [ShellCheck](https://www.shellcheck.net/) says "No issues detected!"

For ref: `man comm` says:

```
       -2     suppress column 2 (lines unique to FILE2)

       -3     suppress column 3 (lines that appear in both files)
```